pages/home.vue

```diff
@@ -3,17 +3,19 @@
         <TimeframeSelection />
         <div class="flex justify-center items-center h-[80vh] w-full p-5">
             <transition name="statistics" mode="out-in">
-                <StatisticsPurplePlaceholder v-if="currentSlide === 0" @click="nextSlide()" />
-                <StatisticsGreenPlaceholder v-else-if="currentSlide === 1" @click="nextSlide()" />
+                <StatisticsPurplePlaceholder v-if="currentSlide === 0" :key="activeTimeframe+'purple'" @click="nextSlide()" />
```

Comment:
irgendwie mag ich dieses system einfach nicht.
(mit static component names und sonst mit v-if, key in @click)

Reply:
Was genau stört dich? was meinst du mit static component names? Das auto-import system von nuxt? stört dich was am `v-if`? stört dich was am `@click`? stört dich was am key-switching um die component bei Timeframe-änderung neu zu rendern?